src/main/java/org/junit$pioneer/jupiter/TempDirectory.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit$pioneer.jupiter;
```

Comment:
Hmmmmm.... is the official package name really going to be `junit$pioneer` with a dollar sign?

Reply:
🤢

Reply:
Thanks for the link. I'll follow up there...

Reply:
@sbrannen I dare you. 😏

Reply:
I'm not going to comment on the $ in the package name but I am wondering about adding ``jupiter``.  Pioneer was supposed to imply that these were (all) jupiter extensions.  I think there would be some benefit in using the package to group together the "rule replacement" extensions though.

Reply:
I was just following the lead of the existing code in this case. There's already a `org.junit$pioneer.vintage` package so a `org.junit$pioneer.jupiter` seemed only logical.

@smoyer64 Are you arguing for a subpackage?

It might be easiest for users to have all extensions in one package so they are easy to discover. On the other hand, a package per extension would allow to have package-private helper classes etc.

Reply:
I agree with @marcphilipp: we should use a `jupiter` subpackage for things specific to JUnit Jupiter.

Reply:
I think it's also worthwhile considering a package per extension since that will make it easier to maintain, especially given that we expect Pioneer to house multiple different types of extensions with varying levels of complexity (_read_: number of helper classes, annotations, interfaces, etc.).